samples/bluetooth/central_uart/src/main.c

```diff
@@ -318,6 +327,14 @@ static void gatt_discover(struct bt_conn *conn)
 		return;
 	}
 
+	static struct bt_gatt_exchange_params exchange_params;
```

Comment:
Strange to put this in gatt_discover.

I would call this from connect callback, we don't have to wait for security for this.

Reply:
Don't we also need to synchronize with event generated by this call ?

Reply:
Changed

Reply:
I am not sure if synchronization is necessary, maybe we could simply use the default MTU until it's updated?

Reply:
I think that we can